src/__tests__/__snapshots__/LinkConnect.test.tsx.snap

```diff
@@ -84,6 +85,71 @@ exports[`LinkConnect Component renders correctly when linkToken is provided 1`]
             "https://google.com",
             "https://front-web-platform-dev",
             "https://front-b2b-api-test.azurewebsites.net",
+            "https://applink.robinhood.com",
+            "https://bonfire.robinhood.com",
+            "https://api.stripe.com",
+            "https://js.stripe.com",
+            "https://m.stripe.network",
+            "https://bonfire.robinhood.com",
+            "https://crumbs.robinhood.com",
+            "https://fp.robinhood.com",
+            "https://nummus.robinhood.com",
+            "https://cdn.robinhood.com",
+            "https://fp.robinhood.com",
+            "https://robinhood-api.arkoselabs.com",
+            "https://app.usercentrics.eu",
+            "https://link.trustwallet.com",
+            "https://web.getfront.com",
+            "https://web.meshconnect.com",
+            "cdn.pdst.fm",
+            "www.google-analytics.com",
+            "cdn.robinhood.com",
+            "cdn.pdst.fm",
+            "www.google-analytics.com",
+            "www.googletagmanager.com",
+            "www.google.com",
+            "www.gstatic.com",
+            "tagmanager.google.com",
+            "ssl.google-analytics.com",
+            "connect.facebook.net",
+            "sc-static.net",
+            "d.impactradius-event.com",
+            "www.redditstatic.com",
+            "analytics.tiktok.com",
+            "boards.greenhouse.io",
+            "bat.bing.com",
+            "www.googleadservices.com",
+            "static.ads-twitter.com",
+            "s.yimg.com",
+            "usercentrics.eu",
+            "snap.licdn.com",
+            "www.youtube.com",
+            "tr6.snapchat.com",
+            "tr.snapchat.com",
+            "fcm.quick1fr.com",
+            "fonts.googleapis.com",
+            "images.robinhood.com",
+            "stats.g.doubleclick.net",
+            "i.ytimg.com",
+            "ctfassets.net",
+            "ssl.gstatic.com",
+            "www.facebook.com",
+            "googleads.g.doubleclick.net",
+            "pixel.pointmediatracker.com",
+            "bidr.io",
+            "alb.reddit.com",
+            "analytics.twitter.com",
+            "t.co",
+            "sp.analytics.yahoo.com",
+            "cdn.blisspointmedia.com",
+            "px.ads.linkedin.com",
+            "api.instagram.com",
+            "robinhood.com",
+            "apollo.rhinternal.net",
+            "sentry.io",
+            "contentful.com",
+            "robinhood://",
```

Comment:
Why are you adding wallet native links here?

Reply:
It is needed. otherway it will not work on android

Reply:
We don't even use native link in our flow, so I don't understand why would it be needed?

Reply:
If I remove it then android will not work and will not show RH web. only if I add that then it start working.
RH is redirecting to robinhood app after opening applink.robinhood.com